Junk4Dollars-app/Auctions/AuctionListPresenter.swift

```diff
@@ -0,0 +1,19 @@
+public class AuctionListPresenter {
+    public let view: AuctionListView
+    let apiServices = ApiDependencies.apiServices
```

Comment:
Would there be any advantage to injecting the `apiServices` into the constructor of the presenter? This is definitely a question, and not a suggestion. :)